package.json

```diff
@@ -26,49 +26,49 @@
     "arrowParens": "avoid"
   },
   "dependencies": {
-    "@babel/core": "^7.19.3",
-    "@babel/plugin-proposal-decorators": "^7.19.3",
+    "@babel/core": "^7.20.5",
+    "@babel/plugin-proposal-decorators": "^7.20.5",
     "@babel/plugin-proposal-export-default-from": "^7.18.10",
     "@babel/plugin-transform-react-inline-elements": "^7.18.6",
-    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
-    "@enact/dev-utils": "^5.0.2",
-    "@storybook/addon-actions": "^6.5.12",
-    "@storybook/addon-controls": "^6.5.12",
-    "@storybook/addon-docs": "^6.5.12",
-    "@storybook/addon-toolbars": "^6.5.12",
+    "@enact/dev-utils": "^5.1.0",
+    "@storybook/addon-actions": "^6.5.13",
```

Comment:
storybook has now version 6.5.14
Please upate, npm install and push the 2 files again. Sorry :(